src/Umbraco.Web.UI.Client/src/views/webhooks/logs.html

```diff
@@ -16,8 +16,9 @@
           <umb-checkmark
             ng-if="vm.isChecked(log)"
             checked="vm.isChecked(log)"
```

Comment:
@Zeegaan basically it could just just `checked="true"` here, since it would otherwise not render the `<umb-checkmark>` anyway.

Alternatively just extend `log` object with a `checked` property - then it doesn't need to run execute twice for each log 😁

Reply:
Ah you learn something new every day, this front-end thing is hard 🙈 

I like the 2nd approach 💪 

```
vm.logs.forEach(log => {
   isChecked(log);
   formatDatesToLocal(log);
});

function isChecked(log) {
    log.checked = log.statusCode === "OK (200)";
    return log;
} 
```